exchange/__main__.py

```diff
@@ -0,0 +1,104 @@
+from .entities import ExratesRates, User
+from .logger import Log
+from .modules import DBConnector, DBUser, DBUserLog
+
+log = Log()
+localization = Localization()
+db_connection = DBConnector()
+bot = telebot.TeleBot(config.TELEBOT_TOKEN, parse_mode=None)
+
+@bot.message_handler(commands=['start'])
+def on_start(message: telebot.types.Message):
+    bot.reply_to(message, localization.local['start'])
+
+
+@bot.message_handler(commands=['help', 'list'])
+def on_help(message: telebot.types.Message):
+    try:
+        response = requests.get(
+            f'{config.EXCHANGE_API}/exrates/rates', params={'periodicity': 0}
+        )
+    except requests.ConnectionError:
+        bot.reply_to(message, localization.local['errors']['system'])
+    else:
+        rates: list[ExratesRates] = [ExratesRates(**rate) for rate in response.json()]
+        buttons = [
+            [
+                telebot.types.InlineKeyboardButton(
+                    text=rate.Cur_Name, callback_data=rate.Cur_Abbreviation
+                )
+            ]
+            for rate in rates
+        ]
+        keyboard = telebot.types.InlineKeyboardMarkup(buttons)
+        bot.send_message(
+            chat_id=message.chat.id, text=localization.local['select_country'], reply_markup=keyboard
+        )
+
+
+@bot.callback_query_handler(func=lambda message: True)
+@bot.message_handler(func=lambda message: True)
+def current_exrates_rates(message):
+    if isinstance(message, telebot.types.Message):
+        message_handler: telebot.types.Message = message
+        user = User(
+            user_id=message_handler.from_user.id,
+            first_name=message_handler.from_user.first_name,
+            last_name=message_handler.from_user.last_name,
+            html_text=message_handler.html_text,
+            date=message_handler.date,
+            chat_id=message_handler.chat.id,
+        )
+    else:
+        callback_handler: telebot.types.CallbackQuery = message
+        user = User(
+            user_id=callback_handler.from_user.id,
+            first_name=callback_handler.from_user.first_name,
+            last_name=callback_handler.from_user.last_name,
+            html_text=callback_handler.data,
+            date=callback_handler.message.date,
+            chat_id=callback_handler.message.chat.id,
+        )
+    try:
+        log.info('insert user data to DB...')
+        DBUserLog(db_connection).insert(user)
+        DBUser(db_connection).insert(user)      
+    except:
+        log.error(localization.local['errors']['system'])
+        bot.send_message(chat_id=user.chat_id, text=localization.local['errors']['system'])
+    try:
+        response = requests.get(
+            f'{config.EXCHANGE_API}/exrates/rates', params={'periodicity': 0}
+        )
+    except requests.ConnectionError:
+        log.error(localization.local['errors']['system'])
+        bot.send_message(chat_id=user.chat_id, text=localization.local['errors']['system'])
+    else:
```

Comment:
Опционально, но я предпочитаю делать так, чтобы убрать лишний нестинг с `else`
```suggestion
        bot.send_message(chat_id=user.chat_id, text=localization.local['errors']['system'])
        return
```

Reply:
Исправил